sdkv2.0.0/sdkv2.0.0/MojioSDK/MojioClient.swift

```diff
@@ -210,6 +210,30 @@ class MojioClient: NSObject {
         return self
     }
     
+    func query (top : String?, skip : String?, filter : String?, select : String?, orderby : String?) -> Self {
+        
+        self.REQUEST_URL = self.REQUEST_URL + "?"
+        
+        if top != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$top=" + top!
+        }
+        if skip != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$skip=" + skip!
+        }
+        if filter != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$filter=" + filter!
+        }
+        if select != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$select=" + select!
+        }
+        if orderby != nil {
+            self.REQUEST_URL = self.REQUEST_URL + "$orderby=" + orderby!
+        }
+
```

Comment:
Does this work with multiple parameters?  I would normally expect query string parameters to be separated by ampersands (&) when multiple values are included, but wasn't certain the API didn't utilize custom parameter parsing.